include/dlaf/eigensolver/bt_band_to_tridiag/mc.h

```diff
@@ -135,8 +138,131 @@ auto updateE(pika::threads::thread_priority priority, WSender&& tile_w, W2Sender
       pika::execution::experimental::start_detached();
 }
 
+struct Helper {
+  Helper(const SizeType b, const SizeType nrefls, matrix::Distribution dist,
+         const GlobalElementIndex offset)
```

Comment:
Perhaps a more descriptive name can be found?

Reply:
Yes! Open to suggestions! 😄

Reply:
Just some ideas:
TileShapeHelper
TileAccessHelper
TileAccessSpecs
SubAccessSpecs

Reply:
Currently, the namespace enclosing the full implementation (together with helpers) is `dlaf::eigensolver::internal`, which is not specific to this algorithm.

I fully agree with @teonnik about looking for a better naming for `Helper`. In order to do that I think we should think about what the name should do and if it has to identify a specific helper for this algorithm or if it has also to distinguish this helper from other helpers available for other algorithms.

My proposal would be to enclose helpers inside an additional namespace, e.g. `dlaf::eigensolver::internal::bt_tridiag`, so that we can use a name that indicates what the helper does, leaving the job of discriminating it from other algorithms to the additional namespace.

At that point, name proposed by @rasolca are perfectly fine: my personal preference is to have Helper inside the name, so I would go for `TileShapeHelper`/`TileAccessHelper`/`TileSpecHelper`.

Reply:
> My proposal would be to enclose helpers inside an additional namespace, e.g. `dlaf::eigensolver::internal::bt_tridiag`, so that we can use a name that indicates what the helper does, leaving the job of discriminating it from other algorithms to the additional namespace.

Fully agree.